make/deps.mk

```diff
@@ -22,7 +22,6 @@ dep_detergent = git https://github.com/pap/detergent e86dfeded3e4f9f3f9278c6a1ae
 dep_jiffy = hex 0.14.7
 dep_nklib = git https://github.com/NetComposer/nklib
 dep_plists = git https://github.com/essen/plists
-dep_erlydtl = git https://github.com/erlydtl/erlydtl 0.12.0
```

Comment:
cause it appears their master branch has less issues (today's merl issue is due to that)